package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Nov 30 09:04:58 UTC 2016 - mfilka@suse.com
+
+- bnc#1012764
+  - made rpm architecture independent (noarch)
```

Comment:
I've talked with @kobliha and he accepted that for SP3, I'll ask him for confirmation even here.